Tests/test_imageops.py

```diff
@@ -362,3 +363,43 @@ def test_auto_contrast_mask_real_input():
             threshold=2,
             msg="autocontrast without mask pixel incorrect",
         )
+
+
+def test_autocontrast_preserve_gradient():
+    gradient = Image.linear_gradient("L")
+
+    # test with a grayscale gradient that extends to 0,255.
+    # Should be a noop.
+    out = ImageOps.autocontrast(gradient, cutoff=0, preserve_tone=True)
+
+    assert_image_equal(gradient, out)
+
+    # cutoff the top and bottom
+    # autocontrast should make the first and list histogram entries equal
+    # and should be 10% of the image pixels (+-, because integers)
```

Comment:
```suggestion
    # and should be 10% of the image pixels (+-, due to rounding)
```
? This might be more descriptive.

Reply:
Yep